cluster/charts/crossplane/crds/aws/aws.crossplane.io_providers.yaml

```diff
@@ -1,9 +1,9 @@
+
```

Comment:
Is this CRD autogenerated? If not, what are the purpose of these changes?

Reply:
Yes. All CRD's are autogenerated with `make manifests`.

Reply:
that's what I thought, but there is a TODO in the yaml which made me wonder